doc/source/train/user-guides/using-gpus.rst

```diff
@@ -104,6 +104,29 @@ You can get a list of associated devices with :meth:`ray.train.torch.get_devices
     trainer.fit()
 
 
+Setting the GPU type
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Ray Train allows you to specify the accelerator type for each worker.
+This is useful if your model training has some GPU memory constraints that requires a specific type of GPU.
```

Comment:
Users may want to use different accelerator types not only for GPU memory constraints, but also for e.g. compute power, cost efficiency, availability, etc. 

Let's just say `This is useful if you want to use a specific accelerator type for model training.`